style.css

```diff
@@ -0,0 +1,145 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
+    min-height: 100vh;
+    background: #d3d3d3;
+    font-family: 'roboto';
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.container{
+    position: absolute;
+    left: 50%;
+}
+.cover{
+    margin: 0 auto;
+
+    min-width: 400px;
+    max-width: 800px;
+    height: 480px;
+
+  
+    background-image: url(pic.jpg);
+    border-radius: 20px;
+    box-shadow: 5px 10px 20px rgba(0, 0, 0, 0.4);
+    transform-origin: left;
+
+    animation: animate 4s ease-in-out normal forwards;
+
+    position: absolute;
+
+    z-index: 1;
+}
+.cover::before{
+    content: '';
+
+    width: 60px;
+
+    background: radial-gradient(#575450 6px,transparent 7px) repeat-y;
+    background-size: 30px 30px;
+
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+
+    box-sizing: border-box;
+}
+.cover::after{
+    content: '';
+    width: 60px;
+    position: absolute;
+    top: 0;
+    left: -10px;
+    bottom: 0;
+    background: linear-gradient(transparent,transparent 13px,black 14px,black 16px,transparent 17px,transparent 30px) repeat-y;
+    box-sizing: border-box;
+    background-size: 30px 30px;
+}
```

Comment:
Could you group similar properties

Reply:
Ohh , i forgot this one , but I'll do this change soon 👍🏻 @Rehan-Khan-Engineer